test/Datadog.Trace.ClrProfiler.IntegrationTests/LogsInjectionTestBase.cs

```diff
@@ -0,0 +1,141 @@
+// <copyright file="LogsInjectionTestBase.cs" company="Datadog">
+// Unless explicitly stated otherwise all files in this repository are licensed under the Apache 2 License.
+// This product includes software developed at Datadog (https://www.datadoghq.com/). Copyright 2017 Datadog, Inc.
+// </copyright>
+
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Text;
+using System.Threading;
+using System.Threading.Tasks;
+using Datadog.Trace.TestHelpers;
+using FluentAssertions;
+using FluentAssertions.Execution;
+using Xunit.Abstractions;
+
+namespace Datadog.Trace.ClrProfiler.IntegrationTests
+{
+    public abstract class LogsInjectionTestBase : TestHelper
```

Comment:
Could we move the bulk of this to a helper class instead? I'd like to use it in the aspnetcore tests too, but this has different setup around finding the sample name etc and the helpers are static anyway as far as I can see?